.github/workflows/push_dev.workflow.yml

```diff
@@ -5,33 +5,9 @@ on:
     branches:
       - 1-dev
       - 2-dev
+      - 3-dev
 
 jobs:
-  dead-links:
```

Comment:
Why you would remove the dead links checks for dev branches?

Reply:
Because if a PR is merged into the dev branch, it is meant to have no dead links (check is performed on pull-requests). But... You're right! Hotfixes commited directly on dev branches and merge errors can occur! I'll restore it!